main.go

```diff
@@ -269,7 +272,16 @@ func main() {
 		failE(fmt.Errorf("Failed to open Keychain: %w", err))
 	}
 
-	httpClient := retry.NewHTTPClient().StandardClient()
+	retryHTTPClient := retry.NewHTTPClient()
+	retryHTTPClient.CheckRetry = func(ctx context.Context, resp *http.Response, err error) (bool, error) {
+		if resp != nil && resp.StatusCode == http.StatusNotFound {
+			log.Debugf("Received HTTP 404, retrying request...")
+			return true, nil
```

Comment:
how many times will it retry? do we have to count it or that is coming from other part of the package?

Reply:
`retry.NewHTTPClient()` created a default retryablehttp.Client using `retryablehttp.NewClient()`.
The default client does a maximum of 4 retries, and the wait time is exponentially distributed from 1 sec to 30 sec between the attempts.